### PR TITLE
Part1 - Nouns and Verbs

### PR DESCRIPTION
**Pull request by transistasis**
In this pull request:
* Added nouns and verbs to main.cpp

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -1,157 +1,172 @@
...
+//  5) 
+//  these actions would require an array of numerical values
+//  Noun:     statistics library
+//  action 1: the statistics library computes the mean of an array of data
+statistics.mean();
+//  action 2: the statistics library computes the standard deviation of
+//            an array of data
+statistics.standardDeviation();
+//  action 3: the statistics library computes the variance of an array of  //            data
+statistics.variance();
```
Comment (matkatmusic) [inline main.cpp:101]: the action you used here is `computes`, but none of your pseudo-code includes this word. 
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -1,157 +1,172 @@
...
+
+
+//  1)
+//  Noun:      cat
+//  action 1:  the cat meows
+cat.meow();
+//  action 2:  the cat naps
+cat.nap();
+//  action 3:  the cat steals food
+cat.stealFood();
```
Comment (matkatmusic) [inline main.cpp:60]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (transistasis) [reply main.cpp]: The verb `compute` has been added to each verb. 

Also, I didn't know that you could add tick boxes to comments. Very cool!

File: main.cpp (hunk 2 above)
Comment (transistasis) [reply main.cpp]: The noun `cat` has been replaced with the noun `sport bike`. All new items use `camelCase`.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**